core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```diff
@@ -509,14 +510,34 @@ readonly REPO_PREREQ_PACKAGES_DIR="$PACKAGES_DIR/repo-prereqs"
 readonly SCRIPT_DIR="$(dirname $(readlink -f $0))" # want absolute path
 
 # files
-readonly REQUIREMENTS_FILE_PATH="$SCRIPT_DIR/requirements.txt"
 readonly SCRIPT_FILE_NAME=$(basename $0)
 readonly LOG_FILE_NAME=${SCRIPT_FILE_NAME/sh/log}
 readonly LOG_FILE_PATH="$SCRIPT_DIR/$LOG_FILE_NAME"
 readonly YUM_CONFIG_BACKUP_FILE_PATH="$SCRIPT_DIR/${SCRIPT_FILE_NAME}-yum-repos-backup-tmp-do-not-remove.tar"
 readonly CRANE_BIN="$SCRIPT_DIR/crane"
 readonly INSTALLED_PACKAGES_FILE_PATH="$SCRIPT_DIR/${SCRIPT_FILE_NAME}-installed-packages-list-do-not-remove.tmp"
 readonly PID_FILE_PATH=/var/run/${SCRIPT_FILE_NAME/sh/pid}
+readonly ADD_NOARCH_REPOSITORIES="$SCRIPT_DIR/add-repositories-noarch.sh"
+
+#arch
+readonly ARCH=$(uname -m)
+echol "Detected arch: ${ARCH}"
+readonly REQUIREMENTS_FILE_PATH="$SCRIPT_DIR/requirements_$ARCH.txt"
```

Comment:
```suggestion
readonly REQUIREMENTS_FILE_PATH="$SCRIPT_DIR/requirements.$ARCH.txt"
```
For filename of rpm package a dot (rarely hyphen) is used as arch separator, e.g. `docker-ce-19.03.14-3.el7.x86_64.rpm` so maybe it's worth to follow this convention and use dot or hyphen?